lib/transforms/tfliteextension_legalizer.cc

```diff
@@ -190,6 +190,45 @@ static std::vector<Def> ConvertDepthwiseConv2D(const TFLITEExtensionInst* ext,
                                 new_inst, builder);
 }
 
+static std::vector<Def> ConvertSqueeze(const TFLITEExtensionInst* ext,
```

Comment:
Maybe we can factor out the same function in onyx parser? https://github.com/alibaba/heterogeneity-aware-lowering-and-optimization/blob/master/lib/transforms/onnxextension_legalizer.cc

Define it as a templated function in a header file like "extension_legalizer_uils.h" ?

Reply:
squeeze op common function has been refactored